packages/storybook-html/src/docs/introduction/welcome.story.mdx

```diff
@@ -4,11 +4,9 @@ import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
 
 # Introduction
 
-The UQ Design System is a guide to how user interfaces should look, feel, behave and be coded at UQ.
+The UQ Design System is a guide to how user interfaces should look, feel, behave and be coded at UQ. It supports the implementation of the [UQ Brand Guidelines](https://marketing-communication.uq.edu.au/one-brand) into user interfaces.
 
-It includes a guide to the core styles, components, patterns and standards for UQ user interfaces. 
-
-A package of styles and components can be provided on request.
+This site includes a guide to the core styles, components, patterns and standards for UQ user interfaces. A package of styles and components can be provided on request.
```

Comment:
Would we benefit from a glossary (`core styles, components, patterns, and standards`)?